scripts/header.txt

```diff
@@ -0,0 +1,23 @@
+/**
+ * Orbs Staking Smart Contract
+ * 
+ * IMPORTANT:
+ *   Staking must be done only using the contract interface functions.
+ *   Never transfer tokens directly to the contract address.
+ *   Tokens transferred directly to the contract cannot be recovered. 
```

Comment:
This is just stylistic and not necessarily any better:
1.  I would go with CAUTION instead of IMPORTANT
2. I'd go with "Do NOT" instead of Never
3. I'd go with "can never be recovered" instead of "cannot be recovered"

Reply:
Fixed 1,2